packages/host/src/components/Login/hooks.ts

```diff
@@ -71,6 +72,8 @@ export const useLoginForm = (
     setAuthToken(token);
     setMRUCredentials({ username: username, store: store });
     setUser({ id: '', name: username });
+    setStoreId(store?.id ?? '');
```

Comment:
ah yes - I guess this is where we diverged on the store ID a bit. I used the `store` object in `hostContext` - the problem with that approach is that it's not set in the host context when refreshing the page etc like it is for local storage